src/plugins/saved_objects_management/public/lib/parse_query.test.ts

```diff
@@ -53,6 +53,15 @@ describe('getQueryText', () => {
     });
   });
 
+  it('uses unknown fields as part of queryText', () => {
+    const query = Query.parse('tag:tag-1 unknown:worlds some search');
+
+    expect(parseQuery(query, [])).toEqual({
+      queryText: 'some search unknown:worlds',
```

Comment:
is it OK to invert the search?

Reply:
@afharo It looks like the order of terms doesn't matter

<img width="906" alt="image" src="https://github.com/elastic/kibana/assets/908371/7c4ef7d1-072b-441f-a9c5-b07938e231db">

Reply:
FMI, do we support double-quote searches here?